java-lib/src/main/java/com/wavefront/dto/Event.java

```diff
@@ -27,9 +27,9 @@
   @JsonProperty
   private String eventId;
   @JsonProperty
-  private long startTime;
+  private long startMillis;
   @JsonProperty
-  private Long endTime;
+  private Long endMillis;
```

Comment:
Missed this in the prev PR, wondering why one is using `long` the other is using `Long`

Reply:
this was to support ongoing events ingestion where `endTime` can be `null`